Libraries/oneCCL/tutorials/scripts/ci_test.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+
+def runJupyterNotebook(input_notebook_filename, output_notebook_filename, conda_env, fdpath='./'):
+    import nbformat
+    import os
+    from nbconvert.preprocessors import ExecutePreprocessor
+    from nbconvert.preprocessors import CellExecutionError
+    if os.path.isfile(input_notebook_filename) is False:
+        print("No Jupyter notebook found : ",input_notebook_filename)
+    try:
+        with open(input_notebook_filename) as f:
+            nb = nbformat.read(f, as_version=4)
+            ep = ExecutePreprocessor(timeout=6000, kernel_name=conda_env, allow_errors=True)
+            ep.preprocess(nb, {'metadata': {'path': fdpath}})
+            with open(output_notebook_filename, 'w', encoding='utf-8') as f:
+                nbformat.write(nb, f)
+            return 0
+    except CellExecutionError:
+        print("Exception!")
+        return -1
+
+print(__file__)
+runJupyterNotebook(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                'oneCCL_Getting_Started.ipynb'),
```

Comment:
Please update the path of oneCCL_Getting_Started.ipynb here, it's one level up from this script, so this should be:
`os.path.join(os.path.dirname(os.path.realpath(__file__)), **'..'** ,'oneCCL_Getting_Started.ipynb'),  'oneCCL_Getting_Started_result.ipynb', 'base')`

Reply:
@e-zorina 
We assume that CI/CD will run sample.json under oneCCL/tutorials folder, so we run ci_test.py "python scripts/ci_test.py"
https://github.com/intel-ai-tce/oneAPI-samples/blob/oneccl_ci/Libraries/oneCCL/tutorials/sample.json#L18

Therefore, we assume that we are in the same folder as oneCCL_Getting_Started.ipynb.

thanks

Reply:
Hi @louie-tsai, yes, CI runs this sample from another path. Also, you are using os.path.realpath(__file__) here, which is always a place where this script, ci_test.py, is located in. So, I don't think it works from any path.

Thanks,
Kate

Reply:
changed it according to Zorina's feedback

Reply:
@e-zorina @louie-tsai made the change. Just want to confirm this is the change that you requested.